app/assets/javascripts/application.js

```diff
@@ -1,2 +1,4 @@
 //= require turbolinks
-//= require app/utils
+
+import 'app/utils';
+import 'app/pw-toggle';
```

Comment:
would love to get this merged for the demo today; @sbc100, @monfresh -- any additional comments?